### ISSUE TITLE
Inserting images in the post summary using relative path does not work on frontpage

### ISSUE THREAD

**Issue by rexwangcc** [vuepress-reco/vuepress-theme-reco#263]
- [x] I confirm that this is an issue rather than a question.

#### What is actually happening?

First of all, thank you so much for your great work!

I'm trying to insert an image to the summary of a post on the frontpage, if I'm using relative path:

```md
---
title: XXX
date: 2020-10-01 03:19:01
categories:
  - foo
tags:
  - bar
---

![IMG](./img.jpg)

Foo-Bar
<!-- more -->
```
The thumbnail on the frontpage is broken even the post view can resolve the image correctly.

However, putting the image into `.vuepress/public` and using absolute paths work:

```md
---
title: XXX
date: 2020-10-01 03:19:01
categories:
  - foo
tags:
  - bar
---

![IMG](/img.jpg)

Foo-Bar
<!-- more -->
```

This makes the content management of blog assets a bit tricky since I'd like to store all of the images (including the cover) closer to the post markdown files.

(sidenote, footnotes ([^1]) which are supported by the `markdown-it-footnote` plugin cannot be resolved either, but I think it's more like a feature request than a bug)

#### Other relevant information

- Your OS: macOS 10.15.6
- Node.js version: v14.12.0
- VuePress version: 1.5.4
- VuePress-theme-reco version: 1.5.7
- Which package manager did you use for the install(npm/cnpm/yarn)? `yarn`

**Comment by recoluan** [comment]
Hello, I'm sorry to tell you that I tried a little, but it didn't solve your problem. If you have a good solution, please let me know.

I'll try more in the future.

**Comment by rexwangcc** [comment]
> Hello, I'm sorry to tell you that I tried a little, but it didn't solve your problem. If you have a good solution, please let me know.
> 
> I'll try more in the future.

Hi, thank you so much for spending some time on it! I ended up organizing the cover images of my posts in `public` as I described above: `.vuepress/public/covers/*.png` and reference them in posts `![IMG](/covers/*.png)`. It's hacky but works, although it would be harder to maintain as the number of covers grow in the future. I will keep exploring if there's a solution to this. Thanks!
